### PR TITLE
Added documentation for rational resampler

### LINKED ISSUE #3160: Add documentation for Rational Resampler
There is no documentation regarding the usage of the rational resampler block parameters as shown below. It would be helpful if there was more explanation regarding the same.   
Rational_resampler

### PR DESCRIPTION
**Pull request by arualok22**
Fixes #3160 

I have added documentation for the rational resampler block.

### CODE REVIEW

**Comment by rohitsanj** [comment]
@drmpeg Please review

File: gr-filter/grc/filter_rational_resampler_xxx.block.yml [hunk 1]
```diff
@@ -59,4 +59,14 @@ templates:
     callbacks:
     - set_taps(${taps})
 
+documentation: |-
+    This block is used to modify the sampling rate of the incoming signal.
```
[the remark this answers is not in the document]
Comment (arualok22) [reply gr-filter/grc/filter_rational_resampler_xxx.block.yml]: I have updated the documentation to match the one at: https://wiki.gnuradio.org/index.php/Rational_Resampler

**Comment by arualok22** [comment]
@marcusmueller let me know if these changes are alright! :)

**Comment by devnulling** [comment]
@arualok22 Please squash the commits for this PR into a single commit. Also please format the commit message to have a prefix such as:

"docs: Added documentation for rational resampler"

**Comment by marcusmueller** [comment]
@devnulling did that for @arualok22

**Comment by arualok22** [comment]
Thanks @devnulling :)

**Pull request merged**